tests/Flow/__snapshots__/ppsi.spec.ts.snap

```diff
@@ -33,7 +34,6 @@ export function givesAFoo3Obj(): AliasFoo3 {
 /**
  * @flow
  */
-
```

Comment:
Ideally we wouldn't remove this newline.  I wonder if we can somehow use the `loc` of all comments at the top of a file that aren't attached to an import.

Reply:
This isn't deleted exactly, it's the comment below that was injected as a "second" top-of-file comment.

Reply:
It's not actually being removed so much as a comment is being shuffled to be directly trailing this comment.